Source/WebCore/dom/Document.h

```diff
@@ -272,6 +273,7 @@ class DOMTimerHoldingTank;
 
 struct ApplicationManifest;
 struct BoundaryPoint;
+struct CaretPositionFromPointOptions;
 struct CSSParserContext;
```

Comment:
```suggestion
struct CSSParserContext;
struct CaretPositionFromPointOptions;
```
Alphabetical order is case sensitive so we put the capitals first.

Reply:
Yes, I was not sure about this one. Seems not caught by style checker. Fixed.